flow-report/test/util-test.tsx

```diff
@@ -79,8 +79,15 @@ describe('useCurrentLhr', () => {
 });
 
 describe('useDerivedStepNames', () => {
-  it('counts up for each mode', () => {
+  it('ignores provided step name', () => {
```

Comment:
Should we just override with provided step names in `useDerivedStepNames`?

Reply:
Yeah, I think making `FlowResult.Step.name` required is simpler, especially for anyone using the data outside the flow-report.

Reply:
Hmm, are you saying we should add the default step names when constructing the flow result?

Reply:
yea, and I misread your comment as saying the same.